src/RoboFileBase.php

```diff
@@ -216,20 +216,29 @@ public function digipolisUpdateDrupal8($opts = ['config-import' => false])
         $collection = $this->collectionBuilder();
         $collection
             ->taskDrupalConsoleStack('vendor/bin/drupal')
-              ->drupalRootDirectory($this->getConfig()->get('digipolis.root.web'))
-              ->maintenance()
-              ->updateDb();
+            ->drupalRootDirectory($this->getConfig()->get('digipolis.root.web'))
+            ->maintenance();
+
+        // When uninstalling modules inside update hook,
+        // we get "dependency on a non-existent service" exception.
+        // Therefore switch to drush for excecuting update hooks.
+        $collection
+            ->taskExecStack()
+            ->exec('cd ' . $this->getConfig()->get('digipolis.root.web') . ' && ../vendor/bin/drush updb --yes');
```

Comment:
Why not use the --root option for drush instead of cd ...
Or use the drush command stack (https://github.com/boedah/robo-drush)?